users/models.py

```diff
@@ -36,11 +36,19 @@ class Profile(TimestampedModel):
         settings.AUTH_USER_MODEL, on_delete=models.SET_NULL, null=True
     )
     first_name = CharPGPPublicKeyField(_("first name"), max_length=30, blank=True)
+    middle_name = CharPGPPublicKeyField(_("middle name"), max_length=30, blank=True)
     last_name = CharPGPPublicKeyField(_("last name"), max_length=150, blank=True)
+    calling_name = CharPGPPublicKeyField(_("calling name"), max_length=50, blank=True)
     email = EmailPGPPublicKeyField(_("email address"), blank=True)
     phone_number = CharPGPPublicKeyField(_("phone number"), max_length=40, null=False)
+    phone_number_nightly = CharPGPPublicKeyField(
+        _("phone number nightly"), max_length=50, blank=True
+    )
     street_address = CharPGPPublicKeyField(_("street address"), max_length=200)
     date_of_birth = DatePGPPublicKeyField(_("date of birth"))
+    personal_identity_code = CharPGPPublicKeyField(
```

Comment:
I'm assuming this is the "HETU" of a person, as in 120120-1234.

Unless this needs to be names like this, I would recommend using "[national identification number](https://en.wikipedia.org/wiki/National_identification_number)" which is the correct term here.

Reply:
If you want to validate this field, I would also recommend adding https://arthurdejong.org/python-stdnum/ to the project, as that has validation logic for Finnish national identification number.

Reply:
@frwickst: The validation is now added to the model's clean method. The validation will be enabled later in API when the next changes are agreed upon and made to the endpoint.

Reply:
And also the name of the field has changed to the correct one.